README.md

```diff
@@ -327,46 +327,36 @@ Use of this source code is governed by a MIT license that can be found in the [L
 
 ## External sources
 
-In `rv32emu` repository, there are some prebuilt ELF files for testing purpose.
-* `aes.elf` : See [tests/aes.c](tests/aes.c)
-* `captcha.elf` : See [tests/captcha.c](tests/captcha.c)
+There are some prebuilt executables in [rv32emu-prebuilt](https://github.com/sysprog21/rv32emu-prebuilt) for testing and benchmarking purpose.
+The prebuilt executables are built from GNU/GCC v9 and [xPack GCC v14.2.0-1](https://github.com/xpack-dev-tools/gcc-xpack) under Ubuntu-20.04.
+
+To fetch the prebuilt executables, run the following command:
+
+```shell
+$ make build-testbenches
+```
+
+Or, to compile the executables from the source, run:
+
+```shell
+$ make build-testbenches USE_PREBUILT=0 CROSS_CC=<riscv-cross-compiler>
```

Comment:
I would expect more consistent naming scheme. That is,
* `make artifact`: fetch the prebuilt files by default.
* `make artifact ENABLE_PREBUILT=0 CROSS_COMPILE=riscv-none-eabi-`: build everything from scratch.